packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -167,11 +188,32 @@ class AndroidValidator extends DoctorValidator {
         messages.add(ValidationMessage.error('Android license status unknown.'));
         return ValidationResult(ValidationType.partial, messages, statusInfo: sdkVersionText);
     }
-
-    // Success.
     return ValidationResult(ValidationType.installed, messages, statusInfo: sdkVersionText);
   }
 
+  bool _checkJavaVersionNoOutput() {
```

Comment:
Make this a `Future<bool>` so that you can use `processManager.run()`

Reply:
Done, and did the same to the checkJavaVersion method in the base Android validator.